include/nodejs/node_osrm_support.hpp

```diff
@@ -1095,9 +1094,9 @@ argumentsToTileParameters(const Nan::FunctionCallbackInfo<v8::Value> &args, bool
         return tile_parameters_ptr();
     }
 
```

Comment:
I'm not a Node guru, so just curious: is behavior changed here? IIUC calling FromJust() with missing values will just blow the app. Will ` x->Uint32Value();` act same?

Reply:
Hi, thanks for taking a look! So Above, there's a guard which checks that x,y,z hold uint32_t value, so calling it should never cause a problem